tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -339,4 +348,57 @@ public void UnsubscribeFromWidgetUpdates()
     {
         Widget.WidgetUpdated -= HandleWidgetUpdated;
     }
+
+    private void AnnounceWarnings(AdaptiveCard card)
+    {
+        foreach (var element in card.Body)
+        {
+            SearchForWarning(element, false);
+        }
+    }
+
+    private void SearchForWarning(IAdaptiveCardElement element, bool isInsideWarningContainer)
+    {
+        // We are only interested in plain texts. Buttons, Actions, Images
+        // and textboxes are all ignored. Including ActionSets and ImageSets.
+        if (element is AdaptiveTextBlock)

```

Comment:
The new variable name can be added on this line.  `element is AdaptiveTextBlock textBlock`

Reply:
Same for all the other if's